src/app/scenes/address/address.component.html

```diff
@@ -176,10 +176,10 @@ <h1>Explore your block</h1>
                       <td class="uk-text-truncate"><a class="ws-p" routerLink="/tx/{{tx.transaction_hash}}">{{tx.transaction_hash}}</a></td>
                       <td>
                         <ng-container *ngIf="tx.created_at != '0001-01-01T00:00:00Z';else updated_at">
-                          {{tx.created_at| date: 'dd/MM/yyyy HH:mm:ss'}} ({{tx.created_at | timeAgo }})
+                          {{tx.created_at}} ({{tx.created_at | timeAgo }})
```

Comment:
There are a lot of `... != '0001-01-01T00:00:00Z'` checks in the surrounding code. Unless that is some kind of angular trick, that looks like a very brittle comparison which depends on the output format. Is this the standard way of doing that? Or is there another way to check for null or 0 or something?